projects/demo/src/app/demo/pages/chip-page/chip-page.component.html

```diff
@@ -0,0 +1,60 @@
+<pa-demo-page>
+    <pa-demo-title>Chip</pa-demo-title>
+    <pa-demo-description>Small and compact chip used to display information, make selections, filter content or input visual content.
+    </pa-demo-description>
+
+    <pa-demo-examples>
+        <pa-chip noCloseButton>Not closable chip</pa-chip>
+        <pa-chip disabled>Disabled chip</pa-chip>
+        <ng-container *ngFor="let chip of chips">
+            <pa-chip *ngIf="chip.contact" [contact]="chip.contact" (closed)="closeChip(chip.name)">{{chip.name}}</pa-chip>
+            <pa-chip *ngIf="!chip.contact" (closed)="closeChip(chip.name)">{{chip.name}}</pa-chip>
+        </ng-container>
+    </pa-demo-examples>
+    <pa-demo-configuration>
+        <div class="pa-demo-configuration-field">
+            <pa-button (click)="addChip()">Add a chip</pa-button>
+        </div>
+        <div class="pa-demo-configuration-field">
+            <pa-button (click)="addAvatarImageChip()">Add an avatar chip with image</pa-button>
+        </div>
+        <div class="pa-demo-configuration-field">
+            <pa-button (click)="addAvatarChip()">Add an avatar chip without image</pa-button>
+        </div>
```

Comment:
simplify your demo: display directly a chip with text only, one with text + close, one with text+avatar+close. you can make the close button as a configurable checkbox if you want, but keep it simple: it's better to see all examples in one go than to have to click on buttons to display more.

Reply:
I don't see your point... if you can remove chips that's just a good idea to add some and avoids the need of reloading the page, where is the problem ?